src/core_atmosphere/Registry.xml

```diff
@@ -599,11 +599,11 @@
 #endif
 		</stream>
 
-		<stream name="restart"
-                        type="input;output"
-                        filename_template="restart.$Y-$M-$D_$h.$m.$s.nc"
-                        input_interval="initial_only"
-                        output_interval="1_00:00:00"
+		<stream name="restart" 
+                        type="input;output" 
+                        filename_template="restart.$Y-$M-$D_$h.$m.$s.nc" 
+                        input_interval="initial_only" 
+                        output_interval="1_00:00:00" 
```

Comment:
@joeolson42 Joe, why d owe need white spaces at the end of lines?

Reply:
To match NCAR's code